src/lib/services/proposal.ts

```diff
@@ -17,6 +17,24 @@ export const fetchGovDepositParams = (
     .get(`${lcdEndpoint}/cosmos/gov/v1beta1/params/deposit`)
     .then(({ data }) => data.deposit_params);
 
+interface ProposalVotingPeriod {
+  proposal_type: string;
+  voting_period: string;
+}
+
+export interface VotingParams {
+  voting_period: string;
+  proposal_voting_periods: ProposalVotingPeriod[];
+  expedited_voting_period: string;
+}
```

Comment:
Let's use `SnakeToCamelCase` type condition and use snake case for raw data from lcd and camel case for internal use

Ref. https://www.typescriptlang.org/play?#code/C4TwDgpgBAygdgQwNYQCoHsDCCC2EA22AzhADwxQQAewEcAJkVEcAE4CWcA5gHxQC8sSjTqMoAAwAkAb04AzCKyioAvgH0Z8xVACqK8QCgoUAPwSZqmdjDtgCfOwBeZeMjRZcBYmR08e+oygALlgAbgMDUEgoKgFYRBQMbDxCBBJSAHIcTjVOW3Z7NXoIMHQiWzUAJgyeCMjwaFdEjxTvADkIFgh6UlQ+QVRhWgYmdAAjACsIAGNgQLNpQONjAG0AaShOKBQQdDllKDT4tyTPVPSNgDJmNk5eAF0QpvdkrzSIDq6epeXjVHX7j9jDxwr8VIEQqhwgZOLRWHIENNoAAlBAAdwAIiUyrYAAoIVi4JiLAD0JOW2TgRWx5WAITgAFccGNFCt7qCoDgEFRqaVaWpIBx0PQQiwONxwmSKTlqJB6LZurycXSoIzmaz2YFKbk4PlCsU+RVCcB2OhRbcJYFSugcGo5OxWCwQotfvbHcA1GN8OhhebxVwOcZ6OgGV6IGokOx8PhnUDlsHKQgTegqXgEFSEAA3LghMY+-AQdOB35QLjChwoNRw22R6OxksN7ZpehqNITegTJCtogdhD0pks1hx4zgsGBcHg+rRLGG4D4wk4JiCZ6nVrvT60HqozE0vEEom1IA